Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2558,6 +2558,12 @@ static EB_ERRORTYPE VerifySettings(\
 		return_error = EB_ErrorBadParameter;
 	} 
 
+    if((config->encoderColorFormat >= EB_YUV422 || config->encoderBitDepth > 8) && config->profile != 4)
+    {
```

Comment:
I remember main10(420 10bit), the profile can be 1, not 4

Reply:
Yes, this is not correct, 420 10bit should set profile to be main10 or 2